### ISSUE TITLE
Twenty Fifteen: Pinterest boards not showing up on category page 2 & onwards 

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/themes#12]
This was originally reported by @ehti in 4130-wpcom-themes:

User has embedded Pinterest boards in their posts for a certain category. As you scroll down the category page, they show up for each post. 

However, as you reach page 2 on the category page, the boards stop showing up. They're still embedded in the posts, but just don't show up. Probably on other archive pages too. 

Here's a gif where page 2 starts with "Brand Research: Gucci" post. 

![](https://merilab.files.wordpress.com/2016/10/bug.gif)

* User report: 1302114-t
* User's site: https://jenniferkateevans.wordpress.com/category/year-2/year-2-research-file/brand-research/

**Comment by sixhours** [comment]
Has it been confirmed that this is only happening on WP.com? I'm going to test locally to rule out moving this ticket to WP.org.

**Comment by sixhours** [comment]
I've narrowed this down to a conflict between Infinite Scroll (both scroll and click methods) and the Pinterest embed. It happens regardless of theme -- I tested Twenty Fifteen, Dara, and Karuna, with Infinite Scroll enabled and disabled on my Atomic site. All themes had the issue, and it goes away when you completely disable Jetpack's Infinite Scroll. WP.com customers can't do this.

Going to check the Jetpack repo to see if it's been reported already, and open a ticket there if not.
